src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,66 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeptions.InvalidUserAgeException;
+import core.basesyntax.exeptions.InvalidUserDataException;
+import core.basesyntax.exeptions.InvalidUserLoginException;
+import core.basesyntax.exeptions.InvalidUserPasswordException;
+import core.basesyntax.exeptions.NullUserException;
```

Comment:
There is a typo in the package name `exeptions`. It should be `exceptions` to maintain consistency and adhere to standard naming conventions.